docs/develop/create-sso-office-add-ins-nodejs.md

```diff
@@ -66,7 +66,6 @@ This article walks you through the process of enabling single sign-on (SSO) in a
 
 1. In the dialog, check the boxes for the following permissions (some may already be checked by default): 
     * Files.Read.All
```

Comment:
Please change "permissions" to "permission", and we should also remove the plural implication in the parens - i.e. "it might already by checked by default."